lizmap/modules/lizmap/classes/lizmapLogItem.class.php

```diff
@@ -54,18 +54,26 @@ class lizmapLogItem
         'email',
     );
 
-    public function __construct($key)
+    /**
+     * Construct the object, you should use the lizmapLogConfig::getLogItem() method
+     * which will call this constructor
+     *
+     * @param string $key the name of the item
+     * @param array $readConfigPath the array containing the fields of lizmapLogConfig.ini.php
+     */
+    public function __construct($key, $readConfigPath)
```

Comment:
$readConfigPath should be only the content of the section corresponding to the item. It would avoid to check if the section exists etc, since it has already be done into getLogItem().